recipes/sip/bld.bat

```diff
@@ -0,0 +1,8 @@
+%PYTHON% configure.py --sysroot=%PREFIX% --bindir=%LIBRARY_BIN%
```

Comment:
In other platforms `sip.h` is installed in `$PREFIX/include/pythonX.X`. On Windows, the equivalent directory is `%PREFIX%\include`.